examples/js/postprocessing/SMAAPass.js

```diff
@@ -37,7 +37,7 @@ THREE.SMAAPass = function ( width, height ) {
 	this.areaTexture.format = THREE.RGBFormat;
 	this.areaTexture.minFilter = THREE.LinearFilter;
 	this.areaTexture.generateMipmaps = false;
-	this.areaTexture.needsUpdate = true;
+	this.areaTexture.needsUpdate = false;
```

Comment:
I think you can remove this line. Same for the `searchTexture`.

Reply:
I thought needsUpdate is default true :-)
But it looks like it is default undefined. I will change this.